sdk/tests/deqp/functional/gles3/es3fInstancedRenderingTests.js

```diff
@@ -376,7 +376,6 @@ var deMath = framework.delibs.debase.deMath;
         var numVertexAttribArrays = /** @type{number} */ (gl.getParameter(gl.MAX_VERTEX_ATTRIBS));
         for (var idx = 0; idx < numVertexAttribArrays; idx++) {
             gl.disableVertexAttribArray(idx);
-            gl.vertexAttribPointer(idx, 4, gl.FLOAT, gl.FALSE, 0, null);
```

Comment:
Here we set all attributes format to GL_FLOAT by vertexAttribPointer during deinit. 
If some attributes are INT or UINT in shader, and one of them is not used at all (no vertexAttrib API to specify data), so according to the spec, it will fail during drawArray/drawElements and their variants. Because vertexAttrib's type doesn't not match with attrib type in shader. (They don't know vertexAttribPointer call sites are used to deinit variables). 
If no vertexAttribPointer is called during deinit, It is OK. 

Besides, the API vertexAttribPointer here is different from that in native deqp(https://android.googlesource.com/platform/external/deqp/+/master/framework/opengl/gluStateReset.cpp#64). Because in WebGL, the last argument is always an offset, it will specify buffer data to variable in shader. This is not a de-initiation at all.  
However, in native deqp, the last argument in vertexAttribPointer means a NULL pointer because ARRAY_BUFFER is bound to 0 during deinit (https://android.googlesource.com/platform/external/deqp/+/master/framework/opengl/gluStateReset.cpp#52), so the data specified to the variable in shader will be unbind. This is really a de-initiation.